### PR TITLE
NONE: Fixed `DualConnection#CONNECTION_CLOSED_MESSAGE` scope

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: src/main/java/com/atlassian/db/replica/api/DualConnection.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
  * Falls back to the main database if it cannot use a replica.
  */
 public final class DualConnection implements Connection {
-    public static final String CONNECTION_CLOSED_MESSAGE = "This connection has been closed.";
+    private static final String CONNECTION_CLOSED_MESSAGE = "This connection has been closed.";
```
Comment (dagguh) [inline src/main/java/com/atlassian/db/replica/api/DualConnection.java:47]: Ah yes, [my late feedback](https://github.com/atlassian-labs/db-replica/pull/6#discussion_r560155305) 😄
Thanks for the follow-through!

File: src/main/java/com/atlassian/db/replica/api/DualConnection.java [hunk 2]
```diff
@@ -44,7 +44,7 @@
  * Falls back to the main database if it cannot use a replica.
  */
 public final class DualConnection implements Connection {
-    public static final String CONNECTION_CLOSED_MESSAGE = "This connection has been closed.";
```
Comment (dagguh) [inline src/main/java/com/atlassian/db/replica/api/DualConnection.java:47]: Technically it removes part of API. Luckily we're still in 0.x.y SemVer.
Shall we go for 1.0.0 soon? We're gonna [use it on prod, which means it's high time](https://semver.org/#how-do-i-know-when-to-release-100).
We can still make breaking changes afterwards, just gotta communicate it via a major bump.

**Review by dagguh — APPROVED** [review]

File: src/main/java/com/atlassian/db/replica/api/DualConnection.java (hunk 2 above)
Comment (wyrzyk) [reply src/main/java/com/atlassian/db/replica/api/DualConnection.java:47]: It doesn't. The changes were not released ;)

> Shall we go for 1.0.0 soon? 

Anytime

Comment (dagguh) [reply src/main/java/com/atlassian/db/replica/api/DualConnection.java:47]: > The changes were not released ;)

Right!

Comment (dagguh) [reply src/main/java/com/atlassian/db/replica/api/DualConnection.java:47]: > Anytime

Right on, I'll release after #7 merge

**Pull request merged**